src/Responder/FormattedResponder.php

```diff
@@ -149,11 +148,11 @@ protected function assertValid(array $data)
         parent::assertValid($data);
 
         foreach ($data as $formatter => $quality) {
-            if (!is_subclass_of($formatter, AbstractFormatter::class)) {
+            if (!is_subclass_of($formatter, FormatterInterface::class)) {
                 throw new InvalidArgumentException(sprintf(
                     'All formatters in `%s` must implement `%s`',
                     static::class,
-                    AbstractFormatter::class
+                    FormatterInterface::class
                 ));
```

Comment:
Maybe for better readability and reducing line of code to make a separate class for exceptions?